web/styles/search.css

```diff
@@ -2,150 +2,232 @@
     width: 100%;
     height: var(--header-height);
 
-    .navbar-search:not(.expanded) {
-        display: none;
+    .navbar-search {
+        margin: 5.5px 0;
+        border-radius: 5px;
+        position: absolute;
+        color: hsl(0deg 0% 0% / 50%);
```

Comment:
I've seen this color a bunch just in the context from this PR; it sticks out next to a proper CSS variable. Probably not worth clouding the PR here, but putting repeat colors like this behind a variable could be a good follow-up PR.

Reply:
I can only find it once -- can you point me to where else you were seeing it?

Reply:
I must've gotten goofed up and ignored that one instance was from a removed line.